### ISSUE TITLE
Can votes be sold?

### ISSUE THREAD

**Issue by gersh** [DemocracyEarth/paper#229]
I'm confused about the concept of votes. Are these sellable tokens, which can be converted into Bitcoin or cash? When you vote on something do you give up your vote, or does the number of votes you have affect the weight of your vote? Further, would it be possible to subject things to one vote per person?

**Comment by paulamlb** [comment]
Hi! =) There's a FAQ document answering all of these questions, here's the link: https://github.com/DemocracyEarth/paper/blob/master/FAQ.md

**Comment by gersh** [comment]
I see it says you can sell your vote. That is a big deal, but I don't see it in the paper. 

Essentially, you have an oligarchy with basic income.  I don't get why any election with these sort of votes would have legitimacy. How do you justify an outcome of this election vs a one person one vote election?

**Comment by santisiri** [comment]
I disagree with the FAQ since it doesn't make a good job answering that question. We haven't discussed thoroughly how the tokens get traded and this needs to be clearly addressed.

The claim for oligarchy is a bit absurd considering the tokens get distributed by using an Equality variable that stabilizes the allocation for every member of the network to have a same starting line. In that sense we are aiming to make the vote economy as meritocratic as possible (in the real sense of giving a fair starting line for everyone). 

That said: votes are meant to be delegated to others. We are actively working on breeding economic value to them and to prevent vote selling is coercive. Its up to every Organization in the system to determine how votes are to be used inside the organization (not the vote holders). That's where restrictions apply and if you want to make a 'one vote one person' system, its totally doable.

**Comment by gersh** [comment]
I can see you're going for equality of opportunity but not necessarily equality of outcomes. Further, if the votes can be transferred, what prevents inheritance or charismatic leaders convincing people to give them all their votes?

**Comment by herbstephens** [comment]
@gersh ”equality of opportunity”=democracy ... “equality of outcomes”=communism. Maybe I am misunderstanding your point...but what specifically are you going for?

Yes .. votes can be transferred .. as liquidity is a serious issue in democracy.

It is the main job of a politician to “convince people to give them their vote”.  What is your point, otherwise?

**Comment by santisiri** [comment]
i second what you said @herbstephens: equality of outcomes is fascist by any measure.

@gersh check 'quadratic voting' in the paper. it specifically addresses that.

**Comment by gersh** [comment]
The quadratic voting only works if the votes are delegated explicitly through the contract. If the votes are delegated implicitly through an external bot, I don't think you can enforce the constraints.

The problem with buying ownership of votes is a 51% attack or rather cornering the market. If you can manage to buy ownership of a majority of the votes, the votes are no longer valuable for the minority and you can maintain your majority at minimal cost. Corporate has some protections for minority shareholders. Constitutions projects minority rights, but I don't see any protection here.

There is a difference between convincing people to vote their way in one election, and convincing people to permanently sell or give away their vote.

**Comment by marcoht** [comment]
Ok. Now I think I've found the right topic to contribute.
@gersh agree with your point of view. Maybe I'm missing some requirements of the technology that have been used to develop Sovereign, but I'm also concerned with the legitimacy of the current concept.

My view is that for every issue to be discussed and voted, a defined number of votes are to be distributed equally among all participants who are affected by that issue (maybe jurisdiction, location, organization, etc). Each of the participant would receive one vote to use only in that decision. Then it would be "one person one vote" scheme. This vote, if the person has several delegated votes to him/herself would just replicate (I know this word is not appropriate here in this context :)) his vote and count same decision for all delegated votes.

**Comment by paulamlb** [comment]
Hi @gersh  and @marcoht 

Our software is elastic in the sense that it allows for different organizational configurations that can serve a democracy. If the intent is to allow each member to cast one _vote_ per decision, the organization can establish that. 

So for example let's imagine Alice owns 100 _votes_ and she is a member of an organization that is configured to allow each member to cast only one vote per decision. Consequently, within that organization, she will not be able to use her 100 _votes_ at the same time, only one per decision. Now if Bob is also a part of that organization, and he wants to delegate his _vote_ on a specific decision to Alice, she will then be able to use two _votes_, because that second _vote_ came from a sovereign member entitled to weight in on that decision with his _vote_, and he voluntarily chose to delegate it to Alice. 

The great thing about liquid democracy is that if Bob does not agree with the way that Alice used his _vote_, _he can revoke it_ - contrasting with our current democratic models where representation is **frozen** (the sovereign owner of a vote can never revoke for the entire duration of a mandate). With liquid democracy the representative can't force the original owner into a decision he does not agree with.

**Comment by gersh** [comment]
@paulamlb It seems reasonable organizational would have elastic permissions. I think it would be helpful to further specify what rights votes tokens would give. Would you create an organization to govern the Democracy Earth foundation? It seems like it might be reasonable to have Democracy Earth governed by a bicameral system of one person one vote, and one vote token one vote. 

Further, I think there should be some means to prevent the vote tokens from becoming concentrated into too few hands. For example, a high Gini coefficient could automatically trigger higher universal income. 

It also seems like it might be better to have other uses for the tokens. For example, maybe you could charge tokens to create organizations or get things on the ballot. 

Also, I'd propose changing the name from Vote Token to Democracy Earth Token or Democracy Dollar. I think this would clarify its purpose better.  While wealth and political rights may be linked, I don't see how it makes sense to make them equivalent. People can get into debt through many means, but that doesn't mean they should lose their political rights.

**Comment by marcoht** [comment]
Thanks @paulamlb ! Now it got clear to me.
Would someone answer another question? In the case of a organization configured as "one person one vote one token" per issue, what's the advantage of having more tokens than needed to vote on the issues raised (besides the delegated ones)? I'm presuming that for any democratic organization this configuration would be the fairer one.

**Comment by milieuChris** [comment]
Sounds like we have a 5th columnist in here. Using power (whatever the
source of power) to amass voting power is also coercion. This is a
naturally occurring conflicting concern that will need to be addressed.
Delegating voting power is fine. Coercing or taking voting power from
others is not.

Equality of outcomes has nothing to do with Fascism. Nasty work there using
that trigger.

**Comment by marcoht** [comment]
@milieuChris I think that the mechanisms developed in Sovereign are very nice to avoid coercion. For example, the possibility to change the vote if the delegee had a different vote than expected. Neverthless, it's natural (at least in Brazil where I live) to people sell their vote. With time, experience and frequency of issues voted, I think that people will get that selling their vote is not good idea... I REALLY think that Sovereign, with its ability to provide more frequent consultations to people, will teach citizens they are the only ones who can change their reality, by voting.

**Comment by paulamlb** [comment]
@gersh I think the best place to have such debates would be our slack community. we are getting a little off topic here. also, in order to facilitate discussions it would be great if you could develop the arguments for each suggestion more, so we can understand where it is coming from and give it a proper consideration =)

@marcoht it is not for us to tell, but if you want an example a different configuration could be an organization that allocates more decision-making power to its founders and employees then to its investors. that's just one random example, but from a perspective of individual sovereignty the possibilities of fair configurations are endless.  

@milieuChris the ability to revoke + liquidity are good examples of mechanisms that aim to build into the design of our system a protection against both political and economic coercion.

**Comment by herbstephens** [comment]
Just met @gersh in real life (at SF CASH Conference)!  Thanks for the feedback on our paper.  👍
